app/views/layouts/etm/_settings_menu.html.haml

```diff
@@ -27,11 +27,16 @@
       - if has_active_scenario?
         %a.save{:href => new_scenario_path}= t("header.save_scenario")
         %a#reset_scenario{:href => scenario_reset_path}= t("header.reset_scenario")
+
+    - if has_active_scenario?
+      %li.sep
+        = link_to t("header.show_graph"), et_engine_graph_url, target: "_blank"
```

Comment:
The link is relevant because then we allow people to elaborate a bit on the energy graph of their scenario (it's a bit of an expert feature, of course.)